gravitee-apim-console-webui/src/management/api/history-v4/deployment-info/deployment-info.component.scss

```diff
@@ -0,0 +1,56 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+@use '../../../../scss/gio-layout' as gio-layout;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+:host {
+  @include gio-layout.gio-responsive-content-container;
+}
+
+.card {
+  padding: 24px;
+
+  &__header {
+    margin-bottom: 12px;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+
+    &__title {
+      width: 50%;
+      margin-bottom: 0;
+    }
+
+    &__back_button {
+      width: 50%;
+
+      button {
+        position: absolute;
```

Comment:
I'm not sure it's necessary here. absolute should really be avoided. and only used if necessary.

Reply:
Yeah, you're right. It turned out that this part was unnecessary. I just removed it